### ISSUE TITLE
Error creating user with azuresql_aad_user

### ISSUE THREAD

**Issue by ExtelligenceIT** [kristofferlind/terraform-provider-azuresql#7]
```
Terraform will perform the following actions:

  # azuresql_aad_user.principal1 will be created
  + resource "azuresql_aad_user" "principal1" {
      + database = "sqldb-eli-dev-eun-001"
      + id       = (known after apply)
      + name     = "app-elisapi-dev-eun-001"
    }

Plan: 1 to add, 0 to change, 0 to destroy.
azuresql_aad_user.principal1: Creating...
╷
│ Error: Failed to create user
│
│   with azuresql_aad_user.principal1,
│   on mssql_users.tf line 10, in resource "azuresql_aad_user" "principal1":
│   10: resource "azuresql_aad_user" "principal1" {
│
│ mssql: Unclosed quotation mark after the character string '
│   '.
╵
```

Apologies - not a developer, just a scripter :) - but having the above issue. It seems to be thrown from the mssql engine.

Hopefully to be helpful, and If I am looking in the correct place in the code, it seems that there is a an additional ' character in the code (internam/manager/user.go - line 78) that may be the cause of this issue.

**Comment by kristofferlind** [comment]
Thanks! You are absolutely correct, that singlequote should not be there. Current test setup doesn't cover aad_ resources and I haven't actually used aad_user yet. I've fixed that particular issue and created a 0.2.1 release for it, hopefully there aren't more issues to be uncovered in that resource.

If you have or install nix packages you should have everything you need to try changes available with `nix-shell --pure`. `make install` builds it and places it in provider overrides where it can then be tested in your setup.

**Comment by ExtelligenceIT** [comment]
Used with no issues - Thanks for resolving.
